tests/unit/test_config.py

```
@@ -0,0 +1,63 @@
+
+import os
+import sys
+from pathlib import Path
+
+import pytest
+
+os.environ.setdefault("PULUMI_ALLOW_TEST_DEFAULTS", "1")
+
+sys.path.append(str(Path(__file__).resolve().parents[2] / "pulumi"))
+
+from infra.config import _sanitize_bucket_component, central_logging_bucket_name, settings, state_bucket_name_for_repo  # noqa: E402
```

Ruff (0.14.13)

Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)